### PR TITLE
created an awesome new file

### PR DESCRIPTION
**Pull request by h-pham**
please accept my pull request about okc
- [ ] because you want too
- [ ] seriously
- [ ] things to do
- [ ] things to see

### CODE REVIEW

**Comment by ave0rtega** [comment]
adding comment per instruction.  :-)

**Comment by yasink** [comment]
Can you please deploy code to QA ??

**Comment by h-pham** [comment]
it's reopened yasin!!  you can't bring me down..

File: hpham-okc.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### Stuff about OKC
+* Home of the Thunders
```
Comment (h-pham) [inline hpham-okc.md:2]: the thunders will never win a championship

**Pull request merged**